scripts/cli

```diff
@@ -656,6 +656,7 @@ __rvm_parse_args()
             rvm_token=${rvm_token#--}
             rvm_token=${rvm_token//-/_}
             export "rvm_${rvm_token}_flag"=1
+            rvm_ruby_args+=($next_token);
```

Comment:
this does not look good, the options here do not take arguments and `next_token` is what's listed after them, could be anything